File: CSharp Basics/SoftUni Exams/01.Cinema/Cinema.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _01.Cinema
{
    class Cinema
    {
        static void Main(string[] args)
        {
            string projectionType = Console.ReadLine();
            int rows = int.Parse(Console.ReadLine());
            int columns = int.Parse(Console.ReadLine());
            double sum = 0;

            int people = rows * columns;

            if (projectionType == "Premiere")
            {
                sum = 12d * people;
            }
            else if (projectionType == "Normal")
            {
                sum = 7.50d * people;
            }
            else if (projectionType == "Discount")
            {
                sum = 5d * people;
            }

            Console.WriteLine("{0:0.00} leva", sum);
        }
    }
}

File: Java Basics/IntroductionToJava/src/Cinema.java
import java.util.Scanner;


public class Cinema {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		String type = input.next();
		double rows = input.nextDouble();
		double columns = input.nextDouble();
		
		Double viewers = rows * columns;
		if (type.equals("Premiere")) {
			System.out.printf("%.2f leva", viewers * 12.00);
		}
		else if (type.equals("Normal")) {
			System.out.printf("%.2f leva", viewers * 7.5);
		}
		else if (type.equals("Discount")) {
			System.out.printf("%.2f leva", viewers * 5.00);
		}
	}

}
